verify.go

```diff
@@ -96,6 +96,11 @@ func signature(pb map[string]interface{}) []byte {
 		params = append(params, adNetworkId)
 	}
 
+	sourceIdentifier, ok := pb["source-identifier"].(string)
```

Comment:
Order of parameters is different for version 3 and version 4.
We need to check  postback's version before creating the signature.

Reply:
From what I've seen in the document, there are fields that have just been added and removed. I could not see any difference in the order while generating the combined string. Could you give an example of the parameter whose order is changed? Maybe I'm overlooked.

Reply:
Yes you are right, order did not change. 👍